browser/css/jssidebar.css

```diff
@@ -3,6 +3,11 @@
 	text-transform: none !important;
 }
 
+#sidebar-panel {
+	padding: 0px;
+	margin-inline: 8px;
+}
+
```

Comment:
I don't think this is related to RTL. Could you add this to a different commit with commit message about why/what fixes it?